Fix regexp that matches mtr output lines with TEST 'variant' ... [ fail ].

The regexp did not accept tests or variants with chars like '/' and ','.

--- mtrlogobserver.py.orig	2015-03-17 12:25:35.845100920 +0100
+++ mtrlogobserver.py	2015-03-17 14:21:07.336002643 +0100
@@ -82,8 +82,8 @@
     the Waterfall page. It also passes the information to methods that can be
     overridden in a subclass to do further processing on the information."""
 
-    _line_re = re.compile(r"^([-._0-9a-zA-z]+)( '[-_ a-zA-Z]+')?\s+(w[0-9]+\s+)?\[ (fail|pass) \]\s*(.*)$")
-    _line_re2 = re.compile(r"^[-._0-9a-zA-z]+( '[-_ a-zA-Z]+')?\s+(w[0-9]+\s+)?\[ [-a-z]+ \]")
+    _line_re = re.compile(r"^([-+/.,_0-9a-zA-z]+)( '[-+/.,_ 0-9a-zA-Z]+')?\s+(w[0-9]+\s+)?\[ (fail|pass) \]\s*(.*)$")
+    _line_re2 = re.compile(r"^[-+/.,_0-9a-zA-z]+( '[-+/.,_ 0-9a-zA-Z]+')?\s+(w[0-9]+\s+)?\[ [-a-z]+ \]")
     _line_re3 = re.compile(r"^\*\*\*Warnings generated in error logs during shutdown after running tests: (.*)")
     _line_re4 = re.compile(r"^The servers were restarted [0-9]+ times$")
     _line_re5 = re.compile(r"^Only\s+[0-9]+\s+of\s+[0-9]+\s+completed.$")
